WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/promptslist/compose/BloggingPromptsListItem.kt

```diff
@@ -0,0 +1,118 @@
+package org.wordpress.android.ui.bloggingprompts.promptslist.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.LocalContentColor
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.sp
+import org.wordpress.android.R
+import org.wordpress.android.ui.bloggingprompts.promptslist.model.BloggingPromptsListItemModel
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.compose.unit.FontSize
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.ui.compose.utils.asString
+import org.wordpress.android.ui.utils.UiString.UiStringPluralRes
+import org.wordpress.android.util.LocaleManager
+import java.text.SimpleDateFormat
+
+private const val MONTH_DAY_FORMAT = "MMM d"
+private val answeredColor = Color(0xFF008A20)
```

Comment:
Thinking of reusability/avoiding duplicate code, I was wondering if it would make sense to have this color extracted to a "common" place 🤔
Somewhere in `org.wordpress.android.ui.compose.theme`, maybe? WDYT?

Reply:
I thought about it and noticed we don't have our own `Color` object. Usually, I like creating an object to hold all static references to colors, do you think this could be useful? Maybe something like:

```
object AppColor {
    @Stable
    val Green50 = Color(0xFF008A20)
    // other colors
}
```